TTS/tts/utils/synthesis.py

```diff
@@ -295,7 +295,12 @@ def transfer_voice(
         reference_d_vector = embedding_to_torch(reference_d_vector, cuda=use_cuda)
 
     # load reference_wav audio
-    reference_wav = embedding_to_torch(model.ap.load_wav(reference_wav, sr=model.ap.sample_rate), cuda=use_cuda)
+    reference_wav = embedding_to_torch(
+        model.ap.load_wav(
+            reference_wav, sr=model.args.encoder_sample_rate if model.args.encoder_sample_rate else model.ap.sample_rate
```

Comment:
Not all models have model.args. (Tacotron for example do not have it). So you will need to verify if the model have the attribute "args".

Reply:
Yeah but all models that have VC have model args (only VITS), and the new standard for new models is to have args no?

Reply:
Also true. So it is your call if you want to add the "check" or not.